### PR TITLE
Callback Requests

### PR DESCRIPTION
**Pull request by alex-suspicious**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Other platforms(including the Editor) have not been tested or/and may not work!
 
 ## Installation
 Copy this url: 
-```https://github.com/itisnajim/SocketIOUnity.git```
+```https://github.com/alex-suspicious/SocketIOUnityCallbacks```
```
Comment (itisnajim) [inline README.md:21]: bring it back to https://github.com/itisnajim/SocketIOUnity.git

File: Runtime/SocketIOUnity.cs [hunk 2]
```diff
@@ -81,6 +81,11 @@ public void Emit(string eventName, params object[] data)
         EmitAsync(eventName, data).ContinueWith(t => {});
     }
 
+    public void Emit(string eventName, Action<SocketIOResponse> ack, params object[] data)
+    {
+        EmitAsync(eventName, CancellationToken.None, ack, data).ContinueWith(t => {});
```
Comment (itisnajim) [inline Runtime/SocketIOUnity.cs:86]: LGTM 👍

File: README.md (hunk 1 above)
Comment (itisnajim) [inline README.md:21]: bring this back to https://github.com/itisnajim/SocketIOUnity.git, so i pull this request

Comment (alex-suspicious) [reply README.md]: Done :)

**Pull request merged**